### PR TITLE
24 add toast component

### PR DESCRIPTION
**Pull request by ruthvik129**

### CODE REVIEW

File: packages/opub-ui/src/components/Toast/Toast.module.scss [hunk 1]
```diff
@@ -0,0 +1,135 @@
+@import '../../../styles/common';
+@import '../Button//Button.module.scss';
```
Comment (PixeledCode) [inline packages/opub-ui/src/components/Toast/Toast.module.scss:2]: do we need this import?

File: packages/opub-ui/src/components/Toast/Toast.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Toast } from './Toast';
+import { Button } from '../Button';
+import { useState } from 'react';
+
+/**
+ * Toast Description
```
Comment (PixeledCode) [inline packages/opub-ui/src/components/Toast/Toast.stories.tsx:7]: Add description and reference link to the story

**Review by PixeledCode — CHANGES_REQUESTED** [review]

File: packages/opub-ui/src/components/Toast/Toast.module.scss (hunk 1 above)
Comment (ruthvik129) [reply packages/opub-ui/src/components/Toast/Toast.module.scss]: Removed the imports

File: packages/opub-ui/src/components/Toast/Toast.stories.tsx (hunk 2 above)
Comment (ruthvik129) [reply packages/opub-ui/src/components/Toast/Toast.stories.tsx:7]: Added stories with description

File: packages/opub-ui/src/components/Toast/Toast.module.scss [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    hsl(206 22% 7% / 20%) 0px 10px 20px -15px;
+  padding: var(--space-3);
+  display: grid;
+  grid-template-areas: 'title action close' '. . .';
+  grid-template-columns: auto max-content;
+  column-gap: var(--space-2);
+  align-items: center;
+}
+.ToastRoot[data-state='open'] {
+  animation: slideIn 150ms cubic-bezier(0.16, 1, 0.3, 1);
```
Comment (PixeledCode) [inline packages/opub-ui/src/components/Toast/Toast.module.scss:30]: use the easing that are in tokens

File: packages/opub-ui/src/components/Toast/Toast.module.scss [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+  margin: 0;
+  list-style: none;
+  z-index: 1;
+  outline: none;
+}
+
+.ToastRoot {
+  background-color: var(--background-invert);
+  border-radius: var(--border-radius-1);
+  box-shadow: hsl(206 22% 7% / 35%) 0px 10px 38px -10px,
```
Comment (PixeledCode) [inline packages/opub-ui/src/components/Toast/Toast.module.scss:20]: use box shadow from tokens

**Review by PixeledCode — CHANGES_REQUESTED** [review]

File: packages/opub-ui/src/components/Toast/Toast.module.scss (hunk 4 above)
Comment (ruthvik129) [reply packages/opub-ui/src/components/Toast/Toast.module.scss]: added from tokens

File: packages/opub-ui/src/components/Toast/Toast.module.scss (hunk 3 above)
Comment (ruthvik129) [reply packages/opub-ui/src/components/Toast/Toast.module.scss]: added from tokens

**Review by PixeledCode — APPROVED** [review]

**Review by PixeledCode — APPROVED** [review]

**Pull request merged**